getAlltext.py

```diff
@@ -1,102 +1,98 @@
-            if number in word:
-                is_word = False
-                break
-        if is_word == True:
-            new_word_list.append(word)
-        new_word_list.sort()
-    return new_word_list
+        output: list [strings]
+        '''
+        new_word_list = []
+        numbers = ["0","1","2","3","4","5","6","7","8","9"]
+        p = re.compile("(\.|\!|\,|\?|\:|\;|\)|\(|\\|\/|\'|\")")
+        for word in self.text:
+            is_word = True
+            word = p.sub(" ", word)
+            word = word.lower()
+            for number in numbers:
+                if number in word:
+                    is_word = False
+                    break
+            if is_word == True:
+                new_word_list.append(word)
+            new_word_list.sort()
+        return new_word_list
 
-def countWords(words_list):
-    '''
-    input: list [strings]
+    def countWords(self):
+        '''
+        input: list [strings]
             
         counts number of appearances of each word in the list
     
-    output: dictionary { word : count of appearances }
-    '''
-    words_count = {}
-    for word in words_list:
-        if word in words_count:
-            words_count[word] += 1
-        else:
-            words_count[word] = 1        
-    return words_count
+        output: dictionary { word : count of appearances }
+        '''
+        words_count = {}
+        for word in self.text:
+            if word in words_count:
+                words_count[word] += 1
+            else:
+                words_count[word] = 1
+        return words_count
 
-def top5words(countWords_dict):
-    '''
-    input: dictionary genereted by "countWords" function
+    def top5words(self):
+        '''
+        input: dictionary genereted by "countWords" function
     
-    output: list of 5 keys(words) with biggest value
+        output: list of 5 keys(words) with biggest value
     
-    NOT FULLY FUNCTIONAL
-    '''
-    top_num = 1
-    top_word = ""
-    keys = countWords_dict.keys()
-    for key in keys:
-        if len(key) > 3 and key != 'the':  # banned words    
-            if countWords_dict[key] > top_num:
-                top_num = countWords_dict[key]
-                top_word = key
-    return top_word
+        NOT FULLY FUNCTIONAL
+        '''
+        words = self.countWords()
+        top_num = 0
+        top_word = ""
+        keys = words.keys()
+        for key in keys:
+            if len(key) > 3 and key != 'the':  # banned words
+                if words[key] > top_num:
+                    top_num = words[key]
+                    top_word = key
+        return top_word
 
-def collect_words(address):
-    '''
-    input: address of existing website
-    
-    returns tuple (domain name, dictionary { words on site : number of appearences }) 
-    '''
-    words = standarizeWords(splitToWord(getAlltext(address)))
 
-    return words
\ No newline at end of file
+
+    # def collect_words():
```

Comment:
uncomment this function and if it works with all fixes this PR can be merged to master